From 56640e72dfa80c5c9a67f9b2e17f05f505afeee6 Mon Sep 17 00:00:00 2001
From: WeiYong Bi <bivvy.bi@rock-chips.com>
Date: Thu, 13 Jul 2017 09:46:02 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3368: fix compatible string for lvds
 node

Replace "rockchip,rk33xx-lvds" with "rockchip,rk3368-lvds"

Change-Id: I065de5d994f167129591025e4251944478fa43e7
Signed-off-by: WeiYong Bi <bivvy.bi@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index 310a29e4b661..76bfed57a998 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -1458,7 +1458,7 @@
 	};
 
 	lvds: lvds@ff968000 {
-		compatible = "rockchip,rk33xx-lvds";
+		compatible = "rockchip,rk3368-lvds";
 		reg = <0x0 0xff968000 0x0 0x4000>, <0x0 0xff9600a0 0x0 0x20>;
 		reg-names = "mipi_lvds_phy", "mipi_lvds_ctl";
 		clocks = <&cru PCLK_DPHYTX0>, <&cru PCLK_MIPI_DSI0>;
-- 
2.35.3

